bin/all_sky_search/pycbc_fit_sngls_split_binned

```diff
@@ -82,6 +83,10 @@ parser.add_argument("--gating-veto-windows", nargs='+',
 parser.add_argument("--stat-fit-threshold", type=float, required=True,
                     help="Only fit triggers with statistic value above this "
                          "threshold. Required")
+parser.add_argument("--plot-lower-stat-limit", type=float, required=True,
+                    help="Plot triggers down to this value. Setting this too"
+                         "low will lead to huge memory usage. Perhaps stick"
+                         "with 5.5 or larger.")
```

Comment:
Well, this is true for full scale runs of a large template bank, obviously not for the example settings.  Suggest eg "will incur huge memory usage in a full search: to avoid this choose 5.5 or larger"

Reply:
I don't want to have a default here, I want to be a required option (as different rankings/search sizes etc. may need different values). I've added the words Tom suggests to the help message.